generate_secureblue_iso.sh

```diff
@@ -12,97 +13,76 @@ function is_yes {
     esac
 }
 
-desktop_options=("kinoite" "cinnamon" "bluefin" "silverblue" "sericea" "wayblue-wayfire" "wayblue-sway" "wayblue-river" "wayblue-hyprland")
-desktop_options_asus=("silverblue" "kinoite")
+# Define image configurations
+declare -A image_configs=(
+    ["server"]="Server"
+    ["server-zfs"]="Server"
+    ["silverblue"]="Silverblue"
+    ["kinoite"]="Kinoite"
+    ["bluefin"]="Silverblue:dx"
+    ["sericea"]="Sericea"
+    ["wayblue-wayfire"]="Sericea"
+    ["wayblue-sway"]="Sericea"
```

Comment:
instead of "sericea", shouldn't this just be the empty string? same for cinnamon and others

Reply:
For images like `wayblue-sway`, `bluefin`, and others, I selected the most similar variants based on my understanding of their intended functionality. While exploring the [documentation](https://github.com/JasonN3/build-container-installer), I discovered that if no variant is specified, it defaults to using the `server` variant. This default behavior would require users to manually partition the disk, which can be quite inconvenient.

Reply:
@HryshcIlya makes sense, thanks for clarifying!